### ISSUE TITLE
private deprecated warning

### ISSUE THREAD

**Issue by paroj** [doxygen/doxygen#7877]
with v1.8.18

given
```cpp
class Test
{
private:
    /// @deprecated
    void foo();
};
```
yields
```
deprecated:1: warning: expected <dt> tag but found <p> instead!
```
which is annoying if your CI tests for doxygen warnings. Also no hint is given where the warning comes from.

**Comment by albert-github** [comment]
With the version `.8.`8 I can reproduce the problem, but with the current master (1.8.19 (d5916c06fba9667911a19cf658e7c5ab54daf72c)) the problem is not present anymore.

As far as I can see the problem has been solved in the master version on May 10, 2020 with commit f583f20100441118c0793d47ab00d61f308d512a, based on issue #7738 and the pull request #7740 (`todo` and `depricated` use similar strategies).

I think the problem has been fixed but not released.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.19.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
